content/basic-concepts.md

```diff
@@ -8,7 +8,7 @@ metaDescription: "Basic concepts behind sensenet API"
 
 sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
 
-The Content Repository is an [OData](https://www.odata.org/) producer which means your application can consume our [OData](https://www.odata.org/) service to create web and mobile apps or any other type of content based applications. It supports CRUD operations, properties, binary streams, paging options, custom queries and many more.
```

Comment:
For me The Content Repository is like a magic word. What is a Content Repository?

Reply:
[https://en.wikipedia.org/wiki/Content_repository](https://en.wikipedia.org/wiki/Content_repository) 😁
btw maybe we should change this to sensenet content repository 🤔